lib/pages/home.dart

```diff
@@ -23,62 +20,16 @@ class _HomeState extends State<Home> {
       ),
       body: Stack(
         children: <Widget>[
-          MapWidget(isStatic: false,),
-          FabCircularMenu(
-          child: Container(
-            // leave empty
-          ),
-          ringColor: Colors.white30,
-          ringDiameter: MediaQuery.of(context).size.width * 0.8,
-          fabColor: Theme.of(context).primaryColor,
-          options: <Widget>[
-            IconButton(icon: Icon(
-              Icons.help_outline),
-              onPressed: () { Navigator.pushNamed(context, '/help');},
-              iconSize: 42.0
-            ),
-            IconButton(icon: Icon(
-              Icons.info_outline),
-              onPressed: () { Navigator.pushNamed(context, '/info');},
-              iconSize: 42.0
-            ),
-            IconButton(icon: Icon(
-              Icons.find_replace),
-              onPressed: () { Navigator.pushNamed(context, '/routesetup');},
-              iconSize: 42.0
-            ),
-            IconButton(icon: Icon(
-              Icons.settings),
-              onPressed: () { Navigator.pushNamed(context, '/settings');},
-              iconSize: 42.0
-            ),
-          ],
-        ),
-          Align(
```

Comment:
Please note, that this was a test case you just deleted. It is not necessary to keep this one, but in the future nobody will be able to test this feature.

Reply:
you are totally right, my bad